src/components/Icon/Icon.tsx

```diff
@@ -19,23 +19,157 @@ import InformationIcon from "@/components/icons/InformationIcon";
+import Brackets from "../icons/Brackets";
+import Briefcase from "../icons/Briefcase";
+import Building from "../icons/Building";
+import BurgerMenu from "../icons/BurgerMenu";
+import Cards from "../icons/Cards";
+import CellTower from "../icons/CellTower";
+import CheckInCircle from "../icons/CheckInCircle";
+import ChevronLeft from "../icons/ChevronLeft";
+import Clock from "../icons/Clock";
+import Cloud from "../icons/Cloud";
+import Code from "../icons/Code";
+import CodeInSquare from "../icons/CodeInSquare";
+import Connect from "../icons/Connect";
+import ConnectAlt from "../icons/ConnectAlt";
+import Console from "../icons/Console";
+import Copy from "../icons/Copy";
+import Data from "../icons/Data";
+import Disk from "../icons/Disk";
+import Display from "../icons/Display";
+import Document from "../icons/Document";
+import DotsVertical from "../icons/DotsVertical";
+import DotsHorizontal from "../icons/DotsHorizontal";
+import Email from "../icons/Email";
+import Empty from "../icons/Empty";
+import Fire from "../icons/Fire";
+import Folder from "../icons/Folder";
+import Gift from "../icons/Gift";
+import Home from "../icons/Home";
+import Http from "../icons/Http";
+import Integrations from "../icons/Integrations";
+import LightBulb from "../icons/LightBulb";
+import Lightening from "../icons/Lightening";
+import Loading from "../icons/Loading";
+import Metrics from "../icons/Metrics";
+import MetricsAlt from "../icons/MetricsAlt";
+import Payment from "../icons/Payment";
+import Pencil from "../icons/Pencil";
+import PieChart from "../icons/PieChart";
+import Play from "../icons/Play";
+import Plus from "../icons/Plus";
+import Popout from "../icons/Popout";
+import Question from "../icons/Question";
+import Refresh from "../icons/Refresh";
+import Search from "../icons/Search";
+import Secure from "../icons/Secure";
+import Services from "../icons/Services";
+import Settings from "../icons/Settings";
+import Share from "../icons/Share";
+import SlideIn from "../icons/SlideIn";
+import SlideOut from "../icons/SlideOut";
+import Sparkle from "../icons/Sparkle";
+import Speaker from "../icons/Speaker";
+import Speed from "../icons/Speed";
+import Star from "../icons/Star";
+import Support from "../icons/Support";
+import Table from "../icons/Table";
+import Taxi from "../icons/Taxi";
+import Trash from "../icons/Trash";
+import Upload from "../icons/Upload";
+import Url from "../icons/Url";
 
-const ICONS_MAP = {
+export const ICONS_MAP = {
+  activity: Activity,
   "arrow-down": ArrowDown,
+  "arrow-right": ArrowRight,
+  "arrow-triangle": ArrowTriangle,
+  "arrow-up": ArrowUp,
+  backups: BackUps,
+  blog: Blog,
+  book: Book,
+  brackets: Brackets,
+  briefcase: Briefcase,
+  building: Building,
+  "burger-menu": BurgerMenu,
+  cards: Cards,
+  "cell-tower": CellTower,
   chat: ChatIcon,
   check: CheckIcon,
+  "check-in-circle": CheckInCircle,
   "chevron-down": ChevronDown,
+  "chevron-left": ChevronLeft,
```

Comment:
why did we decide to go with this syntax instead of `chevronLeft` ?

Reply:
Yes
It would be easily readable and we have been using this format
I discussed it with sergio